### ISSUE TITLE
Feature Request: View Files also

### ISSUE THREAD

**Issue by thorus90** [yanncam/KodiWebPortal#15]
In Kodi we have the feature to show files which are not indexed, but it is shown which files are already watched.

I'm using non indexed files for my archiv (series and filmes already watched, but so good I want to keep them).

For series it would be nice to be able to see with KodiWebPortal which episode I watched and which not (after being finished with the serie, I switch to "not watched" in Kodi for the next time.

Thanks for considering, as I guess this will take a bit more efford.

**Comment by yanncam** [comment]
Hello thorus90,

This functionality is already implemented. The "watch / unwatch" status on movies, tvshow, season, episodes is already present.

Plus, a button has been added through KodiWebPortal to change the "watch / unwatch" status directly via KWP, and the change is visible in Kodi.

Check this commit : https://github.com/yanncam/KodiWebPortal/commit/8b7c10e8872f203e99bbfc56b483e7e7f61f95f6

Sincerely,

**Comment by thorus90** [comment]
Hi yanncam,

But this only works for series and movies which are indexed (crawled with i.e. moviedb). For series and movies are not indexed (therefor only reachable via Files in Kodi) are not shown in KWP, at least I cannot see them (perhaps I need to configure sth extra for this?).

**Comment by yanncam** [comment]
Hello @thorus90,

KWP loads movies and tvshows from the database created by Kodi (configured to external database like MySQL or MariaDB). Only scrapped content that are in this DB are displayed in KWP.

For media acceded throught the "Files" menu in Kodi, it's just like an "explorer" inside Kodi, and this content isn't indexed as you said. So KWP can't display it.

You have to index your medias to see them in KWP.

Sincerely,

**Comment by thorus90** [comment]
Yeah but this info is present in the db (in the table `files`,combined with the table `path`), but you are saying no chance you will implement it?

**Comment by boing86** [comment]
Hello

Is it possible to create a third category?

- Movies
- Series
- Browser

This lists only the files which are in viariable NAX_BROWSER_REMOTE_PATH

That is my suggestion.

**Comment by thorus90** [comment]
Unfortunately that would not display the watched status.

**Comment by yanncam** [comment]
As I understand, technicaly it's possible to add a "browser" category which display as a simple "explorer" the content of a specific path.
With data from DB (files table and path table) it will be possible to display a watched / unwatched status, isn't it ?

I precise : I haven't made some test for the moment (sorry, but I'm really busy now).

I'll take a look in the requested feature as soon as possible.

Sincerely,
